_data/simple-icons.json

```
@@ -1120,6 +1120,11 @@
             "hex": "F05032",
             "source": "http://git-scm.com/downloads/logos"
         },
+        {
+            "title": "Gitea",
+            "hex": "609926",
+            "source": "https://gitea.com/"
```

Where on this page can you find the SVG you used?

If it's not there I think the link provided by @PeterShaggyNoble in #1711 is a better alternative. Agreed?

```suggestion
            "source": "https://github.com/go-gitea/gitea/tree/master/assets"
```

---

Agreed on the source URL, as it's the source of both the original SVG and the chosen colour.